tools/deployment/package-helm/templates/query-scheduler-deployment.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: "apps/v1"
+kind: "Deployment"
+metadata:
+  name: {{ include "clp.fullname" . }}-query-scheduler
+  labels:
+    {{- include "clp.labels" . | nindent 4 }}
+    app.kubernetes.io/component: "query-scheduler"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "clp.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: "query-scheduler"
+  template:
+    metadata:
+      labels:
+        {{- include "clp.labels" . | nindent 8 }}
+        app.kubernetes.io/component: "query-scheduler"
+    spec:
+      serviceAccountName: {{ include "clp.fullname" . }}-job-watcher
+      hostname: {{ include "clp.fullname" . }}-query-scheduler
+      subdomain: {{ include "clp.fullname" . }}-query-scheduler
+      terminationGracePeriodSeconds: 10
+      securityContext:
+        runAsUser: {{ .Values.securityContext.firstParty.uid }}
+        runAsGroup: {{ .Values.securityContext.firstParty.gid }}
+        fsGroup: {{ .Values.securityContext.firstParty.gid }}
+      initContainers:
+        - {{- include "clp.waitFor" (dict
+            "root" .
+            "type" "job"
+            "name" "db-table-creator"
+          ) | nindent 10 }}
+        - {{- include "clp.waitFor" (dict
+            "root" .
+            "type" "service"
+            "name" "queue"
+          ) | nindent 10 }}
+        - {{- include "clp.waitFor" (dict
+            "root" .
+            "type" "service"
+            "name" "redis"
+          ) | nindent 10 }}
+      containers:
```

Comment:
readinessProbeTimings and livenessProbeTimings could have been added to enhance healthcheck and also match `docker-compose-all.yaml`. @coderabbitai submit an issue